HOWTO_android.md

```diff
@@ -0,0 +1,637 @@
+<!-- markdownlint-disable MD033 -->
```

Comment:
Isn't a lot of this similar to BUILD?  Could the build instructions be omitted from this?

Reply:
I didn't know if we needed the Android build instructions.  It was a later addition, but I'll strip it.